app/src/main/java/org/android/go/sopt/model/FakeGithubInfoItem.kt

```diff
@@ -0,0 +1,86 @@
+package org.android.go.sopt.model
+
+import com.google.gson.annotations.SerializedName
+
+data class FakeGithubInfoItem(
+    @SerializedName("title") val title: String,
+    val allow_forking: Boolean,
+    val archive_url: String,
+    val archived: Boolean,
+    val assignees_url: String,
+    val blobs_url: String,
+    val branches_url: String,
+    val clone_url: String,
+    val collaborators_url: String,
+    val comments_url: String,
+    val commits_url: String,
+    val compare_url: String,
+    val contents_url: String,
+    val contributors_url: String,
+    val created_at: String,
+    val default_branch: String,
+    val deployments_url: String,
+    val description: String,
+    val disabled: Boolean,
+    val downloads_url: String,
+    val events_url: String,
+    val fork: Boolean,
+    val forks: Int,
+    val forks_count: Int,
+    val forks_url: String,
+    @SerializedName("full_name") val full_name: String,
```

Comment:
```suggestion
    @SerializedName("full_name") val fullName: String,
```
[공식문서의 코틀린 스타일 가이드](https://developer.android.com/kotlin/style-guide#non-constant_names)에 따라 변수명은 lowerCamelCase로 네이밍하는 것이 어떨까요? 
(이미 알고 계실 것 같지만 맥 기준 `⇧F6`으로 한번에 변수명을 변경할 수 있다고 합니다!)